nx-X11/Makefile

```diff
@@ -86,37 +91,38 @@ xmakefile: Imakefile
 # a copy of every rule that might be invoked at top level
 
 clean:
-	    -${MAKE_CMD} ${MAKE_OPTS} $@
-	    find config -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
-	    find lib -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
-	    find programs -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
-	    find include -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
+	-${MAKE_CMD} $@
+	find config -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
+	find lib -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
+	find programs -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
+	find include -type f | egrep '.*/Makefile$$' | while read makefile; do rm -f $$makefile; done
+
```

Comment:
`-delete` is not portable. Agree on `-name Makefile`, though. Also... why does your grep expressions have two dollar signs at the end?!